ddev/src/ddev/cli/clean/__init__.py

```diff
@@ -0,0 +1,22 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+from typing import TYPE_CHECKING
+
+import click
+
+from datadog_checks.dev.subprocess import run_command
```

Comment:
We are investigating why the linter is not checking this package, but this should be removed as it is unused:

```suggestion
```